content/swift/concepts/protocols/terms/hashable/hashable.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Hashable'
+Description: 'Allows types to be hashed into integer values.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Protocols'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`Hashable`** protocol is a fundamental protocol in Swift that allows types to be hashed into integer values, facilitating their use as keys in hash-based collections like dictionaries and sets.
+
+## Syntax
+
+```pseudo
+protocol Hashable {
+    func hash(into hasher: inout Hasher)
```

Comment:
complete this function; should we include hasher.combine() or a comment about it

Reply:
change `hasher` to `myhasher` and adjust in explanation below

Reply:
> complete this function; should we include hasher.combine() or a comment about it

we have hasher.combined() in the example.